src/components/common/Button/index.tsx

```diff
@@ -2,40 +2,27 @@ import React from 'react'
 
 import styled from 'styled-components'
 import StyledButton, { ButtonVariations } from 'styles/common/StyledButton'
+import styles from 'styles/styles'
 
 export interface ButtonBaseProps extends React.ButtonHTMLAttributes<Element> {
   kind?: keyof typeof ButtonVariations
 }
 
 export const ButtonBase = styled(StyledButton)<ButtonBaseProps>`
-  border-radius: 2rem;
+  border-radius: ${styles.borderRadius};
```

Comment:
why do we want to import this from the styles? 
Is because we expect to use always the same border radius and u want to have it in a constant?

I mean, if we really want to all have the same value, ok, but I think the more independent the components are the better. I do see, that if we repeat ourselves a lot, we can have some fragments we reuse, just not sure this is the case.

Reply:
So in the context of a `Button` in my view it makes sense to use a default global value for the `borderRadius`. It is to be assumed that we'd use at most 2 distinct values for the border-radius. Where a lot of them will simply use a default.

This applies to other styled properties too.

Reply:
@anxolin I think for buttons this will and should be the case. For others, I 100% agree we should be more granular and create fragments where we need. In this case I believe it should remain this way, for consistency in the buttons